### DISCUSSION TITLE
Can I write ReactiveUI MAUI apps in csharp?

### DISCUSSION THREAD

**Discussion by NickDarvey** [reactiveui/ReactiveUI#3503] [Q&A]
I'd like to avoid XAML and write my UI in csharp but still be able to build delightful hot-reloading MAUI app. Is that possible with ReactiveUI today?

**Comment by LuohuaRain** [comment]
I think what you mean is `MVU Pattern`

**Comment by ibrahim324** [comment]
You might want to take a look at this: https://github.com/VincentH-Net/CSharpForMarkup
